x-pack/plugins/observability_solution/observability_ai_assistant_app/server/functions/visualize_esql.ts

```diff
@@ -6,18 +6,43 @@
  */
 import { esFieldTypeToKibanaFieldType } from '@kbn/field-types';
 import type { ESQLSearchReponse } from '@kbn/es-types';
+import { validateQuery } from '@kbn/esql-validation-autocomplete';
+import { getAstAndSyntaxErrors } from '@kbn/esql-ast';
 import { ESQL_LATEST_VERSION } from '@kbn/esql-utils';
 import { VisualizeESQLUserIntention } from '@kbn/observability-ai-assistant-plugin/common/functions/visualize_esql';
 import { visualizeESQLFunction } from '../../common/functions/visualize_esql';
 import { FunctionRegistrationParameters } from '.';
 
+const getMessageForLLM = (
```

Comment:
ah sorry I wasn't clear here. What I meant was, `content` for a function response returns structured data. So it can be `{ message: 'a string', errorMessages: myErrorMessages }`. You don't have to parse a string with a regex

Reply:
Ah sorry, I didn't get that. Here is what you want https://github.com/elastic/kibana/pull/179919/commits/f409761c5144985c15ce2d358a84db93bf452a99 right?

Reply:
yes!